index.html

```
@@ -0,0 +1,115 @@
+      name="description"
+      content="Transform your images professionally with advanced compression, format conversion, and optimization. Reduce file sizes by up to 90% while maintaining crystal-clear quality. 100% client-side processing."
+    />
+    <meta
+      name="keywords"
+      content="image compression, image optimization, photo compression, image converter, JPEG PNG WebP, file size reduction, image processing, photo editor"
+    />
+    <meta name="author" content="Abdul Rafay" />
+    <meta name="robots" content="index, follow" />
+
+    <!-- Canonical URL -->
+    <link rel="canonical" href="" />
```

HTMLHint (1.5.0)

[error] The attribute [ href ] of the tag [ link ] must have a value.

(src-not-empty)